myhome/materials/serializers.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework.fields import SerializerMethodField
+from rest_framework.serializers import ModelSerializer
+
+from materials.models import Course, Lesson
+
+
+class CourseSerializer(ModelSerializer):
+
+    class Meta:
+        model = Course
+        fields = '__all__'
+
+
+class LessonSerializer(ModelSerializer):
+
+    class Meta:
+        model = Lesson
+        fields = '__all__'
+
+
+class CourseDetailSerializer(ModelSerializer):
+    count_lessons = SerializerMethodField()
+    lesson = LessonSerializer(many=True, read_only=True)
```

Comment:
Тут все таки отдаются уроки, а не урок (в Meta.fields тоже не забудь поправить)
```suggestion
    lessons = LessonSerializer(many=True, read_only=True)
```